server/src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -1967,27 +1897,34 @@ public void deleteConsumer(
         return returnCerts;
     }
 
-    @ApiOperation(notes = "Retrieves the body of the Content Access Certificate for the Consumer",
-        value = "getContentAccessBody", response = String.class)
-    @ApiResponses({ @ApiResponse(code = 404, message = ""), @ApiResponse(code = 304, message = "") })
-    @GET
-    @Path("{consumer_uuid}/accessible_content")
-    @Produces(MediaType.APPLICATION_JSON)
-    public Response getContentAccessBody(
-        @PathParam("consumer_uuid") @Verify(Consumer.class) String consumerUuid,
-        @HeaderParam("If-Modified-Since") @DefaultValue("Thu, 01 Jan 1970 00:00:00 GMT")
-        @DateFormat({ "EEE, dd MMM yyyy HH:mm:ss z" }) Date since) {
-
+    @Override
+    public Response getContentAccessBody(@Verify(Consumer.class) String consumerUuid, String since) {
```

Comment:
I don't think we can add Pattern constrains on parameter here. Since its a overridden method & breaks the rules of inheritance, when I tried this I got the compiler error something like below - 
```Method parameters do not respect the inheritance rules. In subtypes, no parameter constraints may be declared on overridden or implemented methods, nor may parameters be marked for cascaded validation.```

Reply:
I think this might be a limitation of the Pattern annotation, because it's from the javax.validation package. Have you tried keeping the old annotation (`@DateFormat`) in the exact same way it was applied before, and then modifying `OffsetDateTimeParamConverterProvider` to handle that instead of `@Pattern`?